Practice/kruglov/home_task_4/PyLec4T6.py

```diff
@@ -0,0 +1,16 @@
+# 6 PyLecT6.py
+
+def del_find(el, mtx):
```

Comment:
This will not work if target element is present in more than one column.

Reply:
Fixed